src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -110,7 +110,10 @@ export class ResponsiveTabs implements IResponsiveComponent {
   }
 
   private shouldAddTabsToDropdown(): boolean {
-    return this.isOverflowing(this.tabSection.el) && ResponsiveComponentsUtils.isSmallTabsActivated(this.coveoRoot);
+    return (
+      (this.isOverflowing(this.tabSection.el) || this.tabSection.el.clientWidth === 0) &&
```

Comment:
What if the clientWidth is not zero but the clientHeight is zero? Would we have the same issue?

Same comment for line 147.

Reply:
Yes I will apply some changes

Reply:
Finally the height has no impact